flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "skip-appstream-check": true
```

Comment:
This should be removed. Appdata files must be validated.

Reply:
Fixed.